metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/queries_extractor.py

```diff
@@ -158,6 +162,9 @@ def __init__(
         )
 
         self.structured_report = structured_report
+        self.redundant_run_skip_handler = redundant_run_skip_handler
+
+        self.start_time, self.end_time = self._get_time_window()
```

Comment:
if we are not tracking them already, we should log or add to the report these effective start/end time

Reply:
We log it